src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard-area-chart/dashboard-area-chart.component.ts

```diff
@@ -178,42 +175,75 @@ export class DashboardAreaChartComponent implements OnChanges {
     };
   }
 
-  ngOnChanges(changes: SimpleChanges): void {
-    this.updateChartData(changes);
+  ngOnChanges(): void {
+    this.updateChartData();
   }
 
-  private updateChartData(changes: SimpleChanges): void {
-    this.chartData.dataset[0].label = this.label;
-    this.chartData.dataset[1].label = this.label2;
-    this.setChartTicks();
-    if (changes.data && changes.data.currentValue) {
-      this.data = changes.data.currentValue;
-      this.chartData.dataset[0].data = this.formatData(this.data);
-      [this.currentData, this.currentDataUnits] = this.convertUnits(
-        this.data[this.data.length - 1][1]
-      ).split(' ');
-      [this.maxConvertedValue, this.maxConvertedValueUnits] = this.convertUnits(
-        this.maxValue
-      ).split(' ');
-    }
-    if (changes.data2 && changes.data2.currentValue) {
-      this.data2 = changes.data2.currentValue;
-      this.chartData.dataset[1].data = this.formatData(this.data2);
-      [this.currentData2, this.currentDataUnits2] = this.convertUnits(
-        this.data2[this.data2.length - 1][1]
-      ).split(' ');
-    }
-    if (this.chart) {
-      this.chart.chart.update();
-    }
+  ngAfterViewInit(): void {
+    this.updateChartData();
+  }
+
+  private updateChartData(): void {
+    setTimeout(() => {
+      const colorArray = [
+        this.cssHelper.propertyValue('chart-color-yellow'),
+        this.cssHelper.propertyValue('chart-color-green'),
+        this.cssHelper.propertyValue('chart-color-cyan'),
+        this.cssHelper.propertyValue('chart-color-purple'),
+        this.cssHelper.propertyValue('chart-color-red')
+      ];
```

Comment:
have this outside of the function and also have background color lighter as we are using it.